### ISSUE TITLE
distribute error

### ISSUE THREAD

**Issue by doncat99** [FeatureLabs/featuretools#183]
### Bug/Feature Request Description

```
feature_matrix, feature_defs = ft.dfs(entityset=es,
                                          target_entity="whatever",
                                          agg_primitives=[],
                                          n_jobs=CPUS,
                                          chunk_size=1/CPUS)
```

can anyone explain why the below error occurred?

```
Traceback (most recent call last):
  File "/root/anaconda3/lib/python3.6/site-packages/distributed/comm/tcp.py", line 177, in read
    n_frames = yield stream.read_bytes(8)
  File "/root/anaconda3/lib/python3.6/site-packages/tornado/gen.py", line 1055, in run
    value = future.result()
  File "/root/anaconda3/lib/python3.6/site-packages/tornado/concurrent.py", line 238, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 4, in raise_exc_info
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/anaconda3/lib/python3.6/site-packages/distributed/core.py", line 552, in send_recv_from_rpc
    result = yield send_recv(comm=comm, op=key, **kwargs)
  File "/root/anaconda3/lib/python3.6/site-packages/tornado/gen.py", line 1055, in run
    value = future.result()
  File "/root/anaconda3/lib/python3.6/site-packages/tornado/concurrent.py", line 238, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 4, in raise_exc_info
  File "/root/anaconda3/lib/python3.6/site-packages/tornado/gen.py", line 1063, in run
    yielded = self.gen.throw(*exc_info)
  File "/root/anaconda3/lib/python3.6/site-packages/distributed/core.py", line 432, in send_recv
    response = yield comm.read(deserializers=deserializers)
  File "/root/anaconda3/lib/python3.6/site-packages/tornado/gen.py", line 1055, in run
    value = future.result()
  File "/root/anaconda3/lib/python3.6/site-packages/tornado/concurrent.py", line 238, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 4, in raise_exc_info
  File "/root/anaconda3/lib/python3.6/site-packages/tornado/gen.py", line 1063, in run
    yielded = self.gen.throw(*exc_info)
  File "/root/anaconda3/lib/python3.6/site-packages/distributed/comm/tcp.py", line 198, in read
    convert_stream_closed_error(self, e)
  File "/root/anaconda3/lib/python3.6/site-packages/distributed/comm/tcp.py", line 126, in convert_stream_closed_error
    raise CommClosedError("in %s: %s" % (obj, exc))
distributed.comm.core.CommClosedError: in <closed TCP>: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "data.py", line 165, in <module>
    load_data(start_date, processes)
  File "data.py", line 132, in load_data
    chunk_size=1/CPUS)
  File "/root/anaconda3/lib/python3.6/site-packages/featuretools-0.2.0-py3.6.egg/featuretools/synthesis/dfs.py", line 213, in dfs
    verbose=verbose)
  File "/root/anaconda3/lib/python3.6/site-packages/featuretools-0.2.0-py3.6.egg/featuretools/computational_backends/calculate_featu
re_matrix.py", line 238, in calculate_feature_matrix
    dask_kwargs=dask_kwargs or {})
  File "/root/anaconda3/lib/python3.6/site-packages/featuretools-0.2.0-py3.6.egg/featuretools/computational_backends/calculate_featu
re_matrix.py", line 737, in parallel_calculate_chunks
    client.replicate([_es, _saved_features])
  File "/root/anaconda3/lib/python3.6/site-packages/distributed/client.py", line 2641, in replicate
    branching_factor=branching_factor, **kwargs)
  File "/root/anaconda3/lib/python3.6/site-packages/distributed/client.py", line 652, in sync
    return sync(self.loop, func, *args, **kwargs)
  File "/root/anaconda3/lib/python3.6/site-packages/distributed/utils.py", line 275, in sync
    six.reraise(*error[0])
  File "/root/anaconda3/lib/python3.6/site-packages/six.py", line 693, in reraise
    raise value
  File "/root/anaconda3/lib/python3.6/site-packages/distributed/utils.py", line 260, in f
    result[0] = yield make_coro()
  File "/root/anaconda3/lib/python3.6/site-packages/tornado/gen.py", line 1055, in run
    value = future.result()
  File "/root/anaconda3/lib/python3.6/site-packages/tornado/concurrent.py", line 238, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 4, in raise_exc_info
  File "/root/anaconda3/lib/python3.6/site-packages/tornado/gen.py", line 1063, in run
    yielded = self.gen.throw(*exc_info)
  File "/root/anaconda3/lib/python3.6/site-packages/distributed/client.py", line 2601, in _replicate
    branching_factor=branching_factor)
  File "/root/anaconda3/lib/python3.6/site-packages/tornado/gen.py", line 1055, in run
    value = future.result()
  File "/root/anaconda3/lib/python3.6/site-packages/tornado/concurrent.py", line 238, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 4, in raise_exc_info
  File "/root/anaconda3/lib/python3.6/site-packages/tornado/gen.py", line 1063, in run
    yielded = self.gen.throw(*exc_info)
  File "/root/anaconda3/lib/python3.6/site-packages/distributed/core.py", line 555, in send_recv_from_rpc
    % (e, key,))
distributed.comm.core.CommClosedError: in <closed TCP>: Stream is closed: while trying to call remote method 'replicate'
```

-----
*Issues created here on Github are for bugs or feature requests. For usage questions and questions about errors, please ask on Stack Overflow with the [featuretools](https://stackoverflow.com/questions/tagged/featuretools) tag. Check the [documentation](https://docs.featuretools.com/#get-help) for further guidance on where to ask your question.*

**Comment by rwedge** [comment]
Hi @doncat99,

I'll see if I can help you debug.  The 'replicate' method is the step where copies of the EntitySet and features are sent to each of the workers.  Since copying the EntitySet means more memory use, the error might be related to that.  Are you able to lower the number of n_jobs and see if the error persists?

If you have the bokeh python module installed you can use the distributed `dashboard` to monitor the scheduler and workers.

**Comment by rwedge** [comment]
Hopefully this was resolved by #853, closing for now

Can reopen if problem persists
